app/services/bulk_upload/lettings/year2024/row_parser.rb

```diff
@@ -889,12 +896,15 @@ def validate_owning_org_data_given
   end
 
   def validate_owning_org_permitted
-    if owning_organisation && !bulk_upload.user.organisation.affiliated_stock_owners.include?(owning_organisation)
-      block_log_creation!
+    return unless owning_organisation
 
-      if errors[:field_1].blank?
-        errors.add(:field_1, "You do not have permission to add logs for this owning organisation", category: :setup)
-      end
+    bulk_upload_organisation = bulk_upload.user.support? ? Organisation.find(bulk_upload.organisation_id) : bulk_upload.user.organisation
+    return if bulk_upload_organisation.affiliated_stock_owners.include?(owning_organisation)
```

Comment:
This lines goal is not check if the user is a support user, but that if they are, the owning organisation in the row is affiliated with the organisation this bulk upload was started from (by using organisation id stored on the bulk upload to get the organisation).

If support user's can upload any log, they still shouldn't be able to be given a file by a user which includes logs for organisations that the user is not affiliated with. This check prevents a log in the BU that is not affiliated with the organisation the bulk upload was started from, from also being uploaded at the same time.

Reply:
I'll adjust this to use organisation id for all users and try adding a separate error message to state that the reason is because the owning organisation is not affiliated with the specified organisation.